maps/BlackthornGym1F.asm

```diff
@@ -29,7 +29,7 @@ BlackthornGym1F_MapScripts:
 BlackthornGymClairScript:
 	faceplayer
 	opentext
-	checkflag ENGINE_RISINGBADGE
```

Comment:
it's not immediately clear what's going on here. i would prefer if we could make this clearer, or if we could delete this little conditional branch entirely.

Reply:
Cleaned these up to not use the mt. silver flypoint. The only way to fail to receive the tms is to receive regular items instead (which only happens in full item rando). If this does occur, it will just give the badge a second time (which doesn't do anything and should be harmless).

Reply:
right on, thanks.